doc/dev/crimson/crimson.rst

```diff
@@ -64,6 +65,57 @@ The last causes pools to be created by default with the ``crimson`` flag.
 Crimson pools are restricted to operations supported by Crimson.
 ``Crimson-osd`` won't instantiate PGs from non-Crimson pools.
 
+Configure Crimson with Bluestore
+================================
+
+Deploying Crimson with Bluestore as the back end requires setting
+several configuration values. The required options are
+``crimson_seastar_cpu_cores`` and ``crimson_alien_thread_cpu_cores``.
+
+Bluestore is not a Crimson native `object store backend`_.
```

Comment:
Not relevant to a user.  All the user needs to know is that this config is required for bluestore.

Reply:
I have updated this file, removing the information that was irrelevant to the user.